tasks/bulkutils.py

```diff
@@ -0,0 +1,127 @@
+### Delete this file when CCI supports deletes with where (i.e. Nov 2019)
```

Comment:
This can be updated to use cumulusci.tasks.bulkdata.utils, yeah?

Reply:
Deleted.